yarn-project/aztec-rpc/src/simulator_oracle/index.ts

```diff
@@ -43,12 +43,15 @@ export class SimulatorOracle implements DBOracle {
 
   async getNotes(contractAddress: AztecAddress, storageSlot: Fr) {
     const noteDaos = await this.db.getNoteSpendingInfo(contractAddress, storageSlot);
-    return noteDaos.map(({ contractAddress, storageSlot, nonce, notePreimage, nullifier, index }) => ({
+    ///return noteDaos.map(({ contractAddress, storageSlot, nonce, notePreimage, innerNoteHash, uniqueSiloedNoteHash, siloedNullifier, index }) => ({
+    return noteDaos.map(({ contractAddress, storageSlot, nonce, notePreimage, siloedNullifier, index }) => ({
       contractAddress,
       storageSlot,
       nonce,
       preimage: notePreimage.items,
-      nullifier,
+      //innerNoteHash,
+      //uniqueSiloedNoteHash,
```

Comment:
Intentional placeholders, for readability?

Reply:
removed